content/en/getting_started/synthetics/private_location.md

```diff
@@ -19,19 +19,21 @@ The access to this feature is restricted - if you don't have access, reach out t
 
 ## Overview
 
-Private locations allow you to **monitor internal-facing applications or any private URLs** that aren’t accessible from the public internet. They can also be used to:
+Private locations allow you to **monitor internal-facing applications** or **private URLs** that aren’t accessible from the public internet. 
 
-* **Create custom locations** in areas that are mission-critical to your business.
-* **Verify application performance in your internal CI environment** before you release new features to production with [Synthetic CI test integration][1].
-* **Compare application performance** from both inside and outside your internal network.
+You can also:
 
-Private locations come as Docker containers that you can install wherever makes sense inside of your private network. Once created and installed, you can assign [Synthetic tests][2] to your private location just like you would with any regular managed location.
+- Create custom locations in mission-critical areas of your business.
+- Verify the application performance in your internal CI environment before you release new features to production with [Synthetic CI/CD Testing][1].
+- Compare the application performance from inside and outside your internal network.
 
-Your private location worker pulls your test configurations from Datadog’s servers using HTTPS, executes the test on a schedule or on-demand, and returns the test results to Datadog’s servers. You can then visualize your private locations test results in a completely identical manner to how you would visualize tests running from managed locations:
+Private locations are Docker containers that you can install anywhere inside your private network. Once you've created and installed your private location, you can assign [Synthetic API tests][2] just like you would with a managed location.
```

Comment:
Not only API tests, any time of Synthetic tests - let's clarify that

Reply:
Also let's maybe say something like: Once you've created and installed your private location, you can assign Synthetic tests to your private location and visualize its test results just like you would with a managed location.